src/lint.ts

```diff
@@ -113,6 +118,15 @@ export const createLintCommand = () => {
         .argOptional()
         .default(defaultBranchName),
     )
+    .option(
+      "--compare-to <compare-to>",
+      "the head ref to compare against. Defaults to the current working directory",
```

Comment:
How can a ref default to the current working directory?

Reply:
This is how the argument is documented in the underlying optic diff-all command.